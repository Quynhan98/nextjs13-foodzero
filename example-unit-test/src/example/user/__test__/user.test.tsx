import * as renderer from 'react-test-renderer';
import { User } from '..';

describe('Test User component', () => {
  test('Snapshot', () => {
    const component = renderer.create(<User />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
