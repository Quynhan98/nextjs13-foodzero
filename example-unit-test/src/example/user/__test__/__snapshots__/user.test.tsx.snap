// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test User component Snapshot 1`] = `
<div>
  <div>
    Welcome 
  </div>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</div>
`;
